packages/neuron-wallet/src/controllers/app/menu.ts

```diff
@@ -361,6 +362,18 @@ const updateApplicationMenu = (mainWindow: BrowserWindow | null) => {
           loadTransaction(URL.OfflineSign, json, filePath)
         },
       },
+      {
+        label: t('application-menu.tools.broadcast-transaction'),
+        enabled: hasCurrentWallet && !isXpubWallet,
```

Comment:
`Offline sign` should be disabled when the wallet is `xpub wallet` because it cannot sign a transaction, but `broadcast a transaction` doesn't rely on the ability to sign and can be enabled for all wallets.